constants.properties

```diff
@@ -1,4 +1,4 @@
-gradlePluginsVersion=4.0.3
+gradlePluginsVersion=4.0.4
```

Comment:
You no longer need to bump the version number - that's done by a maintainer now.